Pages/Table.php

```diff
@@ -3005,6 +3005,12 @@ function print_field_value($field, &$row = null) {
                     break;
                 case 'yesno':
                     $field['options'] = Array(1=>'No',2=>'Yes');
+                case 'image':
+                    $return   = '';
```

Comment:
can you remove the extra spaces in front of the = .... it makes it easier to search when it's all consistent